retrieval_service/app/app_test.py

```diff
@@ -250,6 +255,20 @@ def test_search_airports_with_bad_params(m_datastore, app, params):
             terminal="FOO BAR",
             category="FEE",
             hour="BAZ",
+            sunday_start_hour=get_time_mock(),
```

Comment:
I don't understand why this is referencing a fixture as a function, or why we need a function here as opposed to inlining it?